frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -97,6 +100,17 @@ const resetNotificationAndNewRowTag = () => {
     resetNewTag();
 };
 
+onMounted(() => {
+    if (
+        LoginUserState.getUserIdpProvider() == IdpProvider.BCEIDBUSINESS &&
+        LoginUserState.getUserAdminRoleGroups()?.includes(
+            AdminRoleAuthGroup.DelegatedAdmin
+        )
+    ) {
+        showTerms();
```

Comment:
Hi Jota, we only want to check this at one place, which will be right after user login to FAM, same as when we call the get my access api (no need to showTerms every time access to the manage permission screen). Let's try to move this to the correct place. And in the following ticket #1383, we'll call a Backend API to determine if needs to show the terms and conditions. 
<img width="995" alt="Screen Shot 2024-06-26 at 10 57 12 AM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/d0d8b4a0-f007-46dc-af9a-6e425aa77d2a">

Reply:
I'll move the code to this file. Thank you!